anitya/templates/logs.html

```diff
@@ -49,11 +51,13 @@
         </li>
         <li>
             {% if page < total_page %}
-              <a href="{{ url_for('browse_logs') }}?page={{page + 1}}">
+              <a href="{{ url_for('browse_logs', page=page+1, project=project,
+                user=user, from_date=from_date) }}">
                 »
               </a>
             {% else %}
-              <a href="{{ url_for('browse_logs') }}?page={{page}}">
+              <a href="{{ url_for('browse_logs', page=page, project=project,
```

Comment:
and same answer :)
but for this one I could take it down since it won't move the other buttons, but I thought we might as well be consistent and keep them both.